packages/edit-site/src/components/global-styles/preview-iframe.js

```diff
@@ -109,6 +110,15 @@ export default function PreviewIframe( {
 	}, [ styles ] );
 	const isReady = !! width;
 
+	const calculatedStyles = {
```

Comment:
Since this is not a calculation of this vector but a computation of its values (we have if conditions hence steps) maybe:
```suggestion
	const computedStyles = {
```